components/01-atoms/typography/text/_yds-text.scss

```diff
@@ -6,6 +6,95 @@ p {
   p:last-child {
     margin-bottom: 0;
   }
+
+  // Code blocks.
+  pre code {
+    display: block;
+    padding: 1em;
```

Comment:
@laura-johnson This `1em` padding should use a spacing variable. Could you please change this from `padding: 1em` to `padding: var(--size-spacing-5)`?

Reply:
Ok great, thanks Joe! I'll make these changes.